code/modules/food_and_drinks/food/snacks_bread.dm

```diff
@@ -178,6 +178,7 @@
 	bitesize = 3
 	w_class = WEIGHT_CLASS_NORMAL
 	slot_flags = ITEM_SLOT_BACK|ITEM_SLOT_BELT
+	attack_verb = list("hon hon'ed", "je ne sais quoi'ed", "touche'd")
```

Comment:
```suggestion
	attack_verb = list("hon hon'ed", "je ne sais quoi'ed", "touche'd")
	force = 1
```

Reply:
no force no attack message no attack verb

Reply:
I tested it without force and I was able to touche a monkey with a baguette. Are you saying that the receiver of the touche will not see this? Pretty sure the bike horn has a force of 0 and it's functional.